stellarphot/visualization/comparison_functions.py

```diff
@@ -707,11 +717,23 @@ def save_tess_files(self, button=None):
         """
         if self._field_name.value:
             self.tess_field_view()
-            self.iw.save(self._field_name.value, overwrite=True)
+            # Remove output file if it exists
+            if os.path.exists(self._field_name.value) and self.overwrite_outputs:
+                os.remove(self._field_name.value)
+            try:
+                self.iw.save(self._field_name.value)
+            except:
+                print("Error saving full field image.")
 
         if self._zoom_name.value:
             self.tess_field_zoom_view()
-            self.iw.save(self._zoom_name.value, overwrite=True)
+            # Remove output file if it exists
+            if os.path.exists(self._zoom_name.value) and self.overwrite_outputs:
```

Comment:
Since this came up twice should it be moved into a function? Hopefully the need for it will go away soon...

Reply:
Lets see if those astrowidgets guys fix this.